src/store/modules/order/actions.ts

```diff
@@ -627,7 +631,8 @@ const actions: ActionTree<OrderState , RootState> ={
         const total = resp.data.grouped?.orderId?.ngroups;
 
         if(payload.viewIndex && payload.viewIndex > 0) orders = state.packed.list.concat(orders)
-        commit(types.ORDER_PACKED_UPDATED, { orders, total })
+        const packedOrders = await dispatch("fetchGiftCardActivationDetails", { isDetailsPage: false, currentOrders: orders })
```

Comment:
We should fetch gift card details for the currently fetched orders only, instead of fetching gift card details for all the orders (orders from state and current orders).